ai/react.py

```diff
@@ -0,0 +1,19 @@
+import re
+import discord
+from discord.utils import get
+
+
+PATTERN_REACTS = {
+    r'^(\d{4}) :: Code `109` :: .*': 'gooddrone'
+}
+
+
+async def parse_for_reactions(message: discord.Message, message_copy=None) -> bool:
+    '''
+    Look for patterns and react with an emote if one matches.
+    '''
+    for (pattern, emote_name) in PATTERN_REACTS.items():
+        if re.match(pattern, message.content):
+            await message.add_reaction(get(message.guild.emojis, name=emote_name))
+
+    return False
```

Comment:
In this drone's opinion, it may be better for consistency for this module to be made a cog, rather than as it is right now. After all, we also have the AI status as a cog.

Reply:
After having given this some ------- I come to agree with 5890. YAGNI. If we don't need it as a cog, we shouldn't write it as a cog. It may become one later, if more functionality gets added.